web/themes/custom/cloudy/templates/content/page/node--page--full.html.twig

```diff
@@ -20,6 +20,17 @@
           )
         }}
       </div>
+
+      {% set page_path = path("<current>")|render|render %}
```

Comment:
Is the second "render" needed here? I'm curious what it does.

Reply:
Haha, I have no idea! Upon futher inspection, it appears to have been a mistake. Going to remove the duplicative render.